content/v2.0/reference/cli/influxd/run.md

```diff
@@ -42,4 +42,6 @@ influxd run
 | `--session-length`         | TTL in minutes for newly created sessions (default `60`)                               | integer    |
 | `--session-renew-disabled` | Disables automatically extending session TTL on request                                |            |
 | `--store`                  | Data store for REST resources (bolt or memory) (default `bolt`)                        | string     |
+| `--tls-cert`               | Path to TLS certificate file                                                           | string     |
+| `--tls-key`                | Path to TLS private key file                                                           | string     |
```

Comment:
I'd merge the new updates from alpha-19 into this branch just to make sure the flags stay in alphabetical order. I don't know how git will order the new lines in the merge process.

Reply:
Also, this table exists in two places and needs to be updated in both places:

- `content/v2.0/reference/cli/influxd/run.md` _(where you currently have it)_
- `content/v2.0/reference/cli/influxd/_index.md`